### DISCUSSION TITLE
Merge Steps for PR Mergers

### DISCUSSION THREAD

**Discussion by s10singh97** [jtr13/cc21fall1#32] [General]
1. Please check the submission against the criteria mentioned in https://jtr13.github.io/cc21fall1/tutorial-for-pull-request-mergers.html (follow sections [12.2](https://jtr13.github.io/cc21fall1/tutorial-for-pull-request-mergers.html#check-branch) to [12.6](https://jtr13.github.io/cc21fall1/tutorial-for-pull-request-mergers.html#request-changes))
2. Go to the main branch of our project (jtr13/cc21fall1) and open the _bookdown.yml file
3. Copy the entire rmd_files section. It would look something like this 
rmd_files: [
  'index.Rmd', # must be first chapter
  'assignment.Rmd',
  ....,
  ....,
]
4. Open the branch of the submitted PR and open their _bookdown.yml file (as shown in [12.7.1](https://jtr13.github.io/cc21fall1/tutorial-for-pull-request-mergers.html#add-chapter-filename-to-_bookdown.yml-in-prs-branch) upto the third bullet point)
5. Remove their rmd_files: [] section and paste the one which we copied from the main branch of our project.
6. Add the name of the new file in single quotes followed by a comma under the labelled section (eg. Cheatsheets, Tutorials etc).
7. Save the edited version.
8. Come back to the PR.
9. Tag me so that I can review and check the final version.
10. Once I review it, I will tag the assigned person to proceed forward with the merge process.

In case of any doubts, please let me know.

Thanks!

**Comment by jtr13** [comment]
Thanks @s10singh97 ! When you get a chance can you edit https://github.com/jtr13/cc21fall1/blob/main/appendix_pull_request_tutorial.Rmd to reflect the new system?

**Reply by s10singh97** [reply]
Professor, I have edited the `appendix_pull_request_tutorial.Rmd` with new steps. Please let me know if new changes are required.

**Reply by jtr13** [reply]
Looks great! If you tweak the system as you go, please update the instructions.
